index.html

```diff
@@ -105,6 +105,9 @@
                     <a class="list-group-item" href="https://squarewave.github.io">
                         Background Hang Reporting <div class="dashboard-description">View browser hang information collected from Nightly users</div>
                     </a>
+                    <a class="list-group-item" href="histogram-simulator/">
+                        Histogram Simulator <div class="dashboard-description">View Bucket Handling Information</div>
```

Comment:
Fits in nicely with the rest, but the description should include what you can do there. "Simulate histogram bucket widths" would work